content/c-sharp/concepts/comments/comments.md

```diff
@@ -0,0 +1,69 @@
+---
+Title: 'C# Comments'
+Description: 'Explaining different types of C# comments.'
+Subjects: - 'Computer Science'
+Tags: 
+- 'Syntax'
+- 'Comments'
+CatalogContent: 'C#-Course'
+---
+
+## Comments
+
+A **comment** is a piece of text within a program that is not executed. It can be used to provide additional information to aid in understanding the code.
+
+
+### Single-line Comments
```

Comment:
We like for our entries to have Syntax and Example headers, each with a code block and an explanatory sentence. It would be good if you could do this with both the single line comments and the multi line comments. What you have is good for the Example section. The syntax section should use a ```pseudo code block like this:

### Syntax

Single-line comments are created using two consecutive forward slashes. The compiler ignores any text after `//` on the same line:

```pseudo
// Comment goes here
executing code // Comment goes here
```